### ISSUE TITLE
Allow arguments to container.stop()

### ISSUE THREAD

**Issue by markmsmith** [testcontainers/testcontainers-node#28]
### Feature Request
It would be helpful to be able to pass arguments to to `container.stop()` which are propagated to the underlying dockerode `container.stop()`.  This would allow specifying the timeout, `t`, that the container will wait before it is killed:  
https://docs.docker.com/engine/api/v1.37/#operation/ContainerStop

When dealing with an ephemeral container for a test, you often don't care about it shutting down cleanly, since you'll be throwing it away.  In order to achieve this in my tests I have to currently do a hacky (and brittle) workaround of setting the underlying container's default options for stop:  
```ts
(testContainer as any).container.container.defaultOptions.stop.t = 1;
await testContainer.stop();
```

It would be much nicer to instead just call the top-level stop with arguments:
```ts
await testContainer.stop({ t: 1 });
```

**Comment by cristianrgreco** [comment]
Available in 1.3.0.

**Comment by markmsmith** [comment]
Confirmed working in 1.3.1 with the command:
```ts
await testContainer.stop({
  timeout: new Duration(0, TemporalUnit.SECONDS)
});
```
